code-reviews/recipes/javascript-and-typescript.md

```diff
@@ -143,8 +143,35 @@ As an alternative [husky](https://github.com/typicode/husky) can be considered t
 
 ## Code Review Checklist
 
-In addition to the [Code Review Checklist](../process-guidance/reviewer-guidance.md) you should also look for these JavaScript specific code review items
+In addition to the [Code Review Checklist](../process-guidance/reviewer-guidance.md) you should also look for these JavaScript and TypeScript specific code review items.
+
+### Javascript
```

Comment:
Some of the suggestions I did for the C# section apply here too (same exact text).

Reply:
Should all be fixed now, thank you!